### PR TITLE
Fix issue #18 scroll of TableView when too many items

### PR DESCRIPTION
**Pull request by rico237**
I also took the liberty of writing my name as a contributor to the readme.md file ... if u don't like it it's ok but at least accept my code fix so the menu scroll for long list of items and don't when there are few items. :) French_Power !!

### CODE REVIEW

File: DropDownMenu.swift [hunk 1]
```diff
@@ -88,7 +88,12 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		
 		menuView = UITableView(frame: CGRect(origin: CGPoint.zero, size: frame.size))
 		menuView.autoresizingMask = .flexibleWidth
-		menuView.isScrollEnabled = false
+		menuView.isScrollEnabled = true
+        menuView.bounces = false
```
Comment (qmathe) [inline DropDownMenu.swift]: Same indentation issue here

File: DropDownMenu.swift [hunk 2]
```diff
@@ -117,7 +122,18 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		super.layoutSubviews()
         let contentHeight = menuCells.reduce(0) { $0 + $1.rowHeight }
 		
-		menuView.frame.size.height = contentHeight
+        //  Set the right height for tableView
```
Comment (qmathe) [inline DropDownMenu.swift]: Would be be nice to indent it with tabs to match the rest of the code. 

Take note that the `let contentHeight = ` line is incorrectly indented. This file has a few indentation issues, I'll fix them in another commit.

File: DropDownMenu.swift [hunk 3]
```diff
@@ -117,7 +122,18 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
...
+        //
+        //  If the total from the cells is less thant the height of the screen
+        //  then we keep all cells height
+        //  Else we take keep the the height of the screen so we can scroll from all its elements
+        
+        if contentHeight <= newTableViewFrame.size.height {
+            menuView.frame.size.height = contentHeight
+        }else{
```
Comment (qmathe) [inline DropDownMenu.swift]: `} else {` instead of `}else{`

File: DropDownMenu.swift [hunk 4]
```diff
@@ -117,7 +122,18 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
...
+            menuView.frame.size.height = newTableViewFrame.size.height - 20 - 40 // -20 for statusbar & -40 for navigation bar
```
Comment (qmathe) [inline DropDownMenu.swift]: ` - 20 - 40` should be replaced by ` - visibleContentOffset`

The visible content offset is used to indicate the height occupied by decorations such as status bar and navigation bar inside `DropDownMenu.container`, when the menu is visible.

File: DropDownMenu.swift [hunk 5]
```diff
@@ -79,7 +79,7 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		}
 	}
 	open var backgroundAlpha = CGFloat(1)
-	
+    var newTableViewFrame : CGRect
```
Comment (qmathe) [inline DropDownMenu.swift]: Would be be nice to get the same indentation than the previous lines (with tabs)

File: Example/Info.plist [hunk 6]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.8.3</string>
```
Comment (qmathe) [inline Example/Info.plist]: I prefer to avoid any version changes until a new version is ready to be released.

File: README.md [hunk 7]
```diff
@@ -55,3 +55,10 @@ Build DropDownMenuKit framework and drop it into your Xcode project.
 #### Files
 
 Drop DropDownMenu.swift, DropDownMenuCell.swift, DropDownTitleView.swift and DropDownMenuKit.xcassets into your Xcode project.
+
```
Comment (qmathe) [inline README.md]: Would be better to keep README untouched. 

On Github, there is a pane that lists all contributors at the top of each project, so many projects using Github don't maintain a separate list of contributors and I follow the same approach.

**Review by qmathe — CHANGES_REQUESTED** [review]
I just made a few comments. The way the scrolling support is implemented looks good. Thanks!

**Comment by rico237** [comment]
Well I think it's ok now made those indentation fixes (my Xcode add somme indentations issue), they looked perfect on my computer but once pushed to github the files were messy ... :/ so i tried to use a tool for indentation, then finally end up indenting with github code editor :D

**Comment by qmathe** [comment]
Thanks for the pull request! I just merged it.

btw to check the indentation in Xcode, you can use `Show Invisibles` in `Editor` menu. 

For switching between 'tabs' and 'spaces' in Xcode, you can select the project row in the project navigator (the left side pane), then go to `Text Settings` in the inspector (the right side pane).

**Pull request merged**